tabs/tab_data.py

```
@@ -253,56 +254,56 @@ def _():
         is_loading_data.set(False)
         ui.notification_show("All data reset", type="warning")
 
-    # --- 2. Metadata Logic (Simplified with Dynamic UI) ---
+    # --- 2. Metadata Logic ---
     
-    # Update Dropdown list
     @reactive.Effect
     def _update_var_select():
         data = df.get()
         if data is not None:
             cols = ["Select..."] + data.columns.tolist()
```

Ruff (0.14.10)

Consider `["Select...", *data.columns.tolist()]` instead of concatenation

Replace with `["Select...", *data.columns.tolist()]`

(RUF005)